### PR TITLE
#192 Allow jenkins to run as a user other than the default

### LINKED ISSUE #192: Can't override the user that the jenkins process runs as
The role seems to provide the ability to set "jenkins_process_{user,group}", but this only sets the owner of init.groovy.d

I need (and can hopefully supply a PR) to fix the role to set the process user and file ownerships to an overrideable user/group (defaulting to jenkins:jenkins as currently)

### PR DESCRIPTION
**Pull request by nhemingw-xilinx**
This fixes #192

### CODE REVIEW

**Comment by nhemingw-xilinx** [comment]
@geerlingguy, this patch works in my environment and passes the tests in travis.  I don't seem to be able to add any reviewers though.  How can I get it reviewed and merged?

File: tasks/settings.yml [hunk 1]
```diff
@@ -26,8 +26,8 @@
   file:
     path: "{{ jenkins_home }}"
     state: directory
-    owner: jenkins
-    group: jenkins
+    owner: '{{ jenkins_process_user }}'
+    group: '{{ jenkins_process_group }}'
```
Comment (sdd1798) [inline tasks/settings.yml:30]: I think **recurse:yes** is needed. When I am trying this on EC2 without recurse it is failing to change ownership of some directories inside Jenkins home. With recurse enable I am able to setup Jenkins properly.

Comment (geerlingguy) [reply tasks/settings.yml:30]: That would be a separate issue, as this PR doesn't change the behavior of that particular `file` module invocation.

**Comment by nhemingw-xilinx** [comment]
I will not have access to the account that raised this PR from the end of this week, and frankly I've given up trying to make contact as I appear to being ignored.

**Pull request closed without merging**